Podfile

```diff
@@ -19,7 +19,7 @@ plugin 'cocoapods-keys', {
 target 'Habitica' do
 pod 'CRToast', :git => 'https://github.com/cruffenach/CRToast'
 pod 'FontAwesomeIconFactory'
-pod 'RestKit', :git => 'https://github.com/RestKit/RestKit.git', :commit => '13d98d5a6a5e06656ad040013dcae149b7cf8b99'
+pod 'RestKit'
```

Comment:
The RestKit commit referenced in the podfile is actually newer than the 0.27 release (and is still the most recent commit in the development branch) and iirc it fixed issues that happened when the project was built for iOS 10. Could you revert that to the specific commit?